### ISSUE TITLE
Projects Blank Page

### ISSUE THREAD

**Issue by rohkud** [TrustinN/TrustinN.github.io#2]
Same issue as #1 

# Steps to reproduce
## Visit this link
```
https://trustinn.github.io/projects/index.html
```

**Comment by TrustinN** [comment]
Yeah Ive been having the same issue. Wondering if this repo is still maintained

**Comment by rohkud** [comment]
Yeah, I don't think it is being maintained. Hopefully the owner can update it or mark it as deprecated...
